tensorflow/python/ops/image_ops.py

```diff
@@ -686,50 +686,52 @@ def resize_images(images,
   return images
 
 
-def per_image_whitening(image):
-  """Linearly scales `image` to have zero mean and unit norm.
+def per_image_whitening(images):
+  """Linearly scales `images` to have zero mean and unit norm.
 
   This op computes `(x - mean) / adjusted_stddev`, where `mean` is the average
-  of all values in image, and
-  `adjusted_stddev = max(stddev, 1.0/sqrt(image.NumElements()))`.
+  of all values in images, and
+  `adjusted_stddev = max(stddev, 1.0/sqrt(images.NumElements()))`.
 
-  `stddev` is the standard deviation of all values in `image`. It is capped
+  `stddev` is the standard deviation of all values in `images`. It is capped
   away from zero to protect against division by 0 when handling uniform images.
 
   Note that this implementation is limited:
-  *  It only whitens based on the statistics of an individual image.
+  *  It only whitens based on the statistics of the individual images
```

Comment:
Perhaps update this statement?

It only whitens based on the marginal statistics of the supplied images. A single standard deviation and mean is calculated for all images provided as input to this function.